QueryBuilder/Query.cs

```diff
@@ -356,7 +355,7 @@ public object FindVariable(string variable)
         private Dictionary<string, object> BuildDictionaryFromObject(object data, bool considerKeys = false)
         {
 
-            Dictionary<string,object> dictionary = new Dictionary<string, object>();
+            Dictionary<string, object> dictionary = new Dictionary<string, object>();
```

Comment:
same thing with this change

Reply:
the one you linked was still commit from issue 30, not 32 because issue 30 was committed yesterday and issue 32 only is pushed recently today